platform/app/package.json

```diff
@@ -27,7 +27,7 @@
     "build": "node --max_old_space_size=4096 ./../../node_modules/webpack/bin/webpack.js --progress --config .webpack/webpack.pwa.js",
     "clean": "shx rm -rf dist",
     "clean:deep": "yarn run clean && shx rm -rf node_modules",
-    "dev": "cross-env NODE_ENV=development webpack serve --config .webpack/webpack.pwa.js",
+    "dev": "cross-env NODE_ENV=production webpack serve --config .webpack/webpack.pwa.js",
```

Comment:
This line needs to be reverted and a couple more htings fixed if this build succeeds